archetypes/archetypes/src/main/archetype/mp/custom/custom-mp.xml

```diff
@@ -44,6 +44,7 @@
         <model>
             <value key="helidon-test">true</value>
             <value key="mainClass" if="${jpms}">${package}.Main</value>
+            <value key="idea-main-class" if="${jpms}">${package}.Main</value>
```

Comment:
You should be able to just use `mainClass` right ?

Reply:
Right ! I used another mustache value because of the condition but as it is part of the template now, it is not necessary anymore. Thanks !